src/templates/all/.make/rust.mk

```diff
@@ -179,5 +179,9 @@ rust-ut-coverage: check-cargo-registry rust-docker-pull
 		--output-dir coverage/)
 	@sed -e "s/\/usr\/src\/app\///g" -i coverage/lcov.info
 
+release-checklist: check-cargo-registry rust-docker-pull
```

Comment:
Run this through our sanitychecks docker

Reply:
https://github.com/Arrow-air/tf-github/pull/140/commits/4eb31b9268274ae2bca55fc48543d26edcf6324f